test/evergreen.yml

```diff
@@ -2799,6 +2828,26 @@ buildvariants:
     - name: unit-test-with-compile
     - name: fops
 
+- name: macos-1014-cmake
+  display_name: "! OS X 10.14 CMake"
+  run_on:
+  - macos-1014
+  expansions:
+    configure_env_vars: -DCMAKE_C_FLAGS="-ggdb"
+    posix_configure_flags: -DHAVE_DIAGNOSTIC=1 -DENABLE_PYTHON=1 -DENABLE_ZLIB=1 -DENABLE_STRICT=1 -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
+    python_binary: 'python3'
+    smp_command: -j $(sysctl -n hw.logicalcpu)
+    cmake_generator: "Unix Makefiles"
+    make_command: make
+    test_env_vars: DYLD_LIBRARY_PATH=$(pwd)
+    is_cmake_build: true
+  tasks:
+    - name: compile
+      # macos-1012 is a more stable distro that we want to use for our PR testing task
+      distros: macos-1012
```

Comment:
This special 'treatment' was because the compile task in the `macos-1014` builder is part of pull request testing. I think we can remove the lines here to let the task run on `macos-1014` distro (configured in the above `run_on` parameter) as this cmake task is not very time-sensitive.

Reply:
Fair! Removed the lines :)